core/src/main/java/fr/sncf/osrd/train/RollingStock.java

```diff
@@ -183,7 +186,8 @@ public RollingStock(
             double gamma,
             GammaType gammaType,
             TractiveEffortPoint[] tractiveEffortCurve,
-            RJSLoadingGaugeType loadingGaugeType
+            RJSLoadingGaugeType loadingGaugeType,
+            Collection<String> tags
```

Comment:
Tags are not linked to a rolling stock but to a train. This list should be moved to the train schedule request.